docs/platform/howto/_manage-org-vpc-peering-azure.md

```diff
@@ -0,0 +1,426 @@
+---
+title: Manage an organization VPC peering with Microsoft Azure
+sidebar_label: Azure peering
+---
+
+import DeleteOrgPeering from "@site/static/includes/vpc/delete-org-peering.md";
+import ConsoleLabel from "@site/src/components/non-swizzled/ConsoleIcons";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Set up a peering connection between your [Aiven organization VPC](/docs/platform/howto/manage-organization-vpc) and a [Microsoft Azure virtual network](https://learn.microsoft.com/en-us/azure/virtual-network/create-peering-different-subscriptions).
+
+Establishing a peering connection between an Aiven organization VPC and an Azure VNet
+requires creating the peering both from the VPC in Aiven and from the VNet in Azure.
+
+To establish the peering from Aiven to Azure, the Aiven Platform's
+[Active Directory application object](https://learn.microsoft.com/en-us/azure/active-directory/develop/app-objects-and-service-principals)
+needs permissions in your Azure subscription. Because the peering is between different AD
+tenants (the Aiven AD tenant and your Azure AD tenant), another application object is
+needed for your Azure AD tenant to create the peering from Azure to Aiven once granted
+permissions to do so.
```

Comment:
```suggestion
needs permissions in your Azure subscription. Because the peering is between different AD 
tenants (the Aiven AD tenant and your Azure AD tenant), your Azure AD tenant needs another 
application object. Once granted permissions, this object allows peering 
from Azure to Aiven.
```

how about breaking the line for added clarity?